### ISSUE TITLE
Suggestions for better names of the difficulties

### ISSUE THREAD

**Issue by Epyx15** [Uproxide/more-difficulties#1]
Casual -> Medium
Tough  -> Hardest
Cruel -> Extreme
(Note: This are just ideas for names that I think would fit better with the game, if you don't like them feel free to keep it as it is now)

**Comment by Hanifinio** [comment]
Yeah I agree with the Medium and Extreme as they do fit better ig

**Comment by Uproxide** [comment]
I do agree on this one

**Comment by Arieximedes** [comment]
good idea !
💯

**Comment by PowerZGD** [comment]
HORRIBLE IDEA! THIS RUINS THE MOD THE NAMES ARE UNIQUE AND I ALREADY CALL THEM THE NAMES THAT THEY ARE NOW DO NOT CHANGE IT.

**Comment by PowerZGD** [comment]
If this change happens I am going to backup the mod textures cuz this change would absolutely fucking ridiculous. And it is very pointless. Like who the fuck needs this? I know I don't. Also + There the only difficulties starting with C and T. These names are so stupid and look like they were made when on drugs. DO NOT CHANGE THEM.

**Comment by PowerZGD** [comment]
> Casual -> Medium Tough -> Hardest Cruel -> Extreme (Note: This are just ideas for names that I think would fit better with the game, if you don't like them feel free to keep it as it is now) Dude if you want to change them just ask her to make you a texture pack or something is this change would be fucking pointless I feel like whoever made the names wanted them to be more unique not this awful suggestion as they are very unoriginal. Again why the fuck would you need this? CUZ I DON'T NEED IT. IT WOULD BE AN AWFUL CHANGE, SO PLEASE DO NOT CHANGE IT.

**Comment by rubiidev18alt** [comment]
How about "Demanding" for Extreme?
